examples/simple.js

```diff
@@ -16,22 +16,14 @@ let e1 = Experiment.create({
 let e2 = Experiment.create({
 	name: 'buy page price experiment',
 	id: 'a40f09ac',
-	variations: [
-		{ object: 25 },
-		{ object: 35 },
-		{ object: 45 }
-	],
+	variations: [{ object: 25 }, { object: 35 }, { object: 45 }],
```

Comment:
I really don't like this kind of formatting, it is much less clear to me, do you feel the same?

We also need to discuss, maybe we should move to 2 spaces tabs. Even though I personally hate them, it seems like with standardjs they are turning into, well, a standard... WDYT?

Reply:
About the array items strange line breaks, this happens because the default max line width is of 80 characters.
A smaller length will probably cause other changes that you won't like as well.
This is [an open issue](https://github.com/prettier/prettier/issues/2716) of Prettier.
We can fix it by adding `// prettier-ignore` before `e2` definition, but I guess you will not like it as well.
IMHO Prettier is the best formatter out there for javascript and its synchronicity with `eslint` is great. But i agree that this case is disturbing. On the other side, this enforcement (linting + unified formatting on precommit) is important for open source project.
About the 2 spaces tab, I agree about moving to them.